internal/validation/is.go

```diff
@@ -0,0 +1,49 @@
+package validation
+
+import (
+	"net/url"
+	"regexp"
+)
+
+var (
+	isUUIDRegexp  = regexp.MustCompile(`[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}`)
+	isRegionRegex = regexp.MustCompile("^[a-z]{2}-[a-z]{3}$")
+	isZoneRegex   = regexp.MustCompile("^[a-z]{2}-[a-z]{3}-[1-9]$")
```

Comment:
Are uppercase regions/zones valid for API calls?

Reply:
nop